app/views/companies/_company_create_modal.html.haml

```diff
@@ -21,7 +21,7 @@
               .row
                 .col
 
-                  = f.label :company_number, "#{t('companies.show.company_number')}",
+                  = f.label :company_number, "#{t('shf_applications.new.new_company_number')}",
```

Comment:
We should be using the I18n.t _lazy lookup_.  (We haven't been using it for all views, but we should.)
If you're going to change this, then the statement should be `t('.company_number)` and the locale key should be `<lang>.companies.company_create_modal`

The locale entry does not belong under `shf_applications` because the modal file (`_company_create_modal.html.haml` is under the views/**companies** directory.

Reply:
Done.